storage/libradb/src/ledger_store/mod.rs

```diff
@@ -360,6 +360,10 @@ impl LedgerStore {
         cs.batch
             .put::<LedgerInfoSchema>(&ledger_info.epoch(), ledger_info_with_sigs)
     }
+
+    pub fn get_root_hash(&self, version: Version) -> Result<HashValue> {
+        Accumulator::get_root_hash(self, version + 1)
```

Comment:
why the version need + 1

Reply:
The accumulator works on "num of leaves" instead of versions (to deal with empty accumulators)